src/axom/sina/include/CppBridge.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef SINA_CPPBRIDGE_HPP
+#define SINA_CPPBRIDGE_HPP
+
+/**
+ * @file
+ *
+ * This file contains functions which can go away once this library is
+ * migrated to C++14 or later.
+ */
```

Comment:
This is probably a good time to dump this compatibility layer and move to C++14. Axom is already at C++14.